Bell'sInequality43.md

```diff
@@ -13,5 +13,59 @@ That is: an electron has spin, location, and so on, even when it is not being me
 Quantum Mechanics suggests that observation influences reality, and can do so retroactively, a notion that, while strange, is consistent with the evidence. Building on this, Bell demonstrated that local Hidden Variables would lead to contradictions with QM predictions in specific scenarios. Thus, Bell's Theorem builds on EPR, revealing a critical requirement known as Bell's Inequality, which was not immediately apparent until 30 years later.
 
 
+
+## Hidden Variable Theory:
+* Arguement about uncertanity property of Quantum Mechanics
+* Hidden Variable
+   * Investing Quantum Mechanics with Local Realism
+   * Underlying deterministic unknown variable in Quantum Mechanics
+* Bohm's Hidden Variable Theory
+* Local Hidden Variable Theory
+   * Principle that an object is only directly influenced by its immediate surroundings
+* EPR paradox - showed non-locality of Quantum Mechanics
+   * Two Photons that separated so far apart
+   * The measurements od onedetermining the other one's states
+* Local Hidden Variable
+   *  a quantity whose value is presently unknown with local property
+      "Spooky action at a distance"
+## Bell's theorem
+* Proppsed by John Stewart Bell, in the paper that " On the Einstein- Podolsky-Rosen paradox", 1964.
+* A awy of distinguishing experimentally between local hidden variable theories and the predictions of Quantum Mechanics
+* Bell's Inequality - $\displaystyle\rho{\left({a},{c}\right)}-\rho{\left({b},{a}\right)}-\rho{\left({b},{c}\right)}\le{1}$
+   * Inequality that derived from local hidden variable theory
+   * Any Quantum correlations under local hidden variable theory do not satisfy bell's inequalities.
+   * Demonstration by bell test experiments
+* Simple version of a Bell's Inequality
+   * Various possible polarization combinations of the two EPR photon
+
+     <img width="341" alt="image" src="https://github.com/user-attachments/assets/1b7e21a3-99a5-4be8-aa7d-48cc0016f0a8">
```

Comment:
Where is this figure from, does it have a copyright? We cannot add copyrighted figures.

Reply:
I have made another figure by myself. I have removed that. I hope it works.